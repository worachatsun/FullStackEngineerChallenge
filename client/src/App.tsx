import React from 'react';
import RouterCompoment from './components/Router';

const App = () => <RouterCompoment />;

export default App;
