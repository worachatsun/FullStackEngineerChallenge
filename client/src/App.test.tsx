import React from "react";
import renderer from "react-test-renderer";
import App from "./App";

describe("<App />", () => {
    test("should render correctly", () => {
        const render = renderer.create(<App />).toJSON();
        expect(render).toMatchSnapshot();
    });
});
