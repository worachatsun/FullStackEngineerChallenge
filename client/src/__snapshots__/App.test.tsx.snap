// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render correctly 1`] = `
<div
  className="sc-AxhUy jytBcu"
>
  <div
    className="sc-AxgMl zDGTk"
  >
    <div
      className="sc-AxheI gdhAZC"
    >
      <img
        alt="logo"
        src="img_logo.png"
      />
      <div>
        Performance Review
      </div>
    </div>
    <div
      className="sc-Axmtr ehiWYG"
    >
      <div
        className="sc-AxiKw jAHgLp"
      >
        <form
          onSubmit={[Function]}
        >
          <div>
            <label
              htmlFor="Username"
            >
              Username
              :
              <input
                className="sc-AxirZ lmasuR"
                name="username"
                type="text"
              />
            </label>
          </div>
          <div>
            <label
              htmlFor="Password"
            >
              Password
              :
              <input
                className="sc-AxirZ lmasuR"
                name="password"
                type="password"
              />
            </label>
          </div>
          <button
            className="sc-AxjAm fbqdBC"
          >
            Signin
          </button>
        </form>
        <div
          className="sc-AxhCb iLDmSv"
        >
          
        </div>
      </div>
    </div>
  </div>
</div>
`;
